pepper-apis/src/appengine/Housekeeping.yaml

```diff
@@ -0,0 +1,14 @@
+service: housekeeping
+runtime: java11
+instance_class: B4
+# B2 allows background threads and we can use manual scaling
+manual_scaling:
+  instances: 1
+
+entrypoint: java -Xmx820m -Xmx820m -jar Housekeeping.jar
+
+# GAE does not like the "=" character in entrypoint and does not support "." in environment variables
+env_variables:
+  config_file: application.conf
+  ddp_firecloudKeysDir: fc_keys
+
```

Comment:
B4 allows for manual scaling and we can just make sure it is running all the time.